### PR TITLE
Step 7 and styling updates

### PR DESCRIPTION
**Pull request by ecraig12345**
Step 7 updates and add nicer styling for <code>

### CODE REVIEW

File: step2-01/demo/README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ myFunc(...arr);
 
 Spreading an object into props for a React component:
 
-```tsx
+```jsx
```
Comment (ecraig12345) [inline step2-01/demo/README.md]: hljs didn't seem to recognize tsx

File: step2-07/demo/README.md [hunk 2]
```diff
@@ -49,9 +45,9 @@ const NewComponent = connect(
 )(OldComponent);
 ```
 
-The `connect()` function takes in a few functions that maps some portion of the state tree and dispatcher functions as props. It is a **higher order function** meaning that the return value of `connect()` is a function that decorates OldComponents into a NewComponent with all the mapped props.
+`connect()` takes in a few functions that map portions of the state tree and dispatcher functions into props. It is a **higher-order function**, meaning that the return value of `connect()` is a function that decorates `OldComponent` into a `NewComponent` with all the mapped props.
 
-This `mapStateToProps` function selects out portions of the state tree. This function informs the connected view when to re-render based on a shallow comparison from previous state.
+A [`mapStateToProps`](https://react-redux.js.org/api/connect#mapstatetoprops-state-ownprops-object) function selects out portions of the state tree. This function informs the connected view when to re-render based on a shallow comparison from previous state.
```
Comment (ecraig12345) [inline step2-07/demo/README.md]: Might be good to have more detail about this

File: step2-07/demo/README.md [hunk 3]
```diff
@@ -61,7 +57,7 @@ function mapStateToProps(state) {
 }
 ```
 
-The `mapDispatchToProps` are functions that will trigger the action message dispatch mechanism of Redux. It looks like this:
+An optional [`mapDispatchToProps`](https://react-redux.js.org/api/connect#mapdispatchtoprops-object-dispatch-ownprops-object) function will trigger the action message dispatch mechanism of Redux. It looks like this:
```
Comment (ecraig12345) [inline step2-07/demo/README.md]: Same here. Specifically that this one is used to generate callback props.

File: step2-01/demo/README.md (hunk 1 above)
Comment (kenotron) [reply step2-01/demo/README.md]: uuugh!!

**Pull request merged**